.jshintrc

```diff
@@ -0,0 +1,30 @@
+{
+  "nonew": true,
+  "jquery": false,
+  "curly": true,
+  "latedef": true,
+  "unused": true,
+  "noarg": true,
+  "indent": 1,
+  "trailing": true,
+  "white": true,
+  "forin": true,
+  "noempty": true,
+  "quotmark": "single",
+  "eqeqeq": true,
+  "strict": true,
+  "undef": true,
+  "bitwise": true,
```

Comment:
I suspect we'll only use bitwise operations in rdm.js  . It would be safer to disable it globally and then just enable it in the required functions.